bundles/paikkatietoikkuna/admin/admin-layeranalytics/Flyout.js

```diff
@@ -35,19 +37,52 @@ Oskari.clazz.define('Oskari.framework.bundle.admin-layeranalytics.Flyout',
 
             this.updateListing();
         },
+        generateToScaleURL (details) {
+            const stack = details.stack[0];
+            let toScaleURL = '/?coord=' + stack.x + '_' + stack.y;
```

Comment:
Add `"&zoomLevel=" + stack.z` to get the same zoom level as where the problem occured.